src/main/java/Duke.java

```diff
@@ -1,10 +1,103 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static final String INITIALISING = "initialising";
+    public static final int MAX_LIST_SIZE = 100;
+    private static Task[] list = new Task[MAX_LIST_SIZE];
+    private static int listSize = 0;
+
+    public static void addToList(Task task) {
+        list[listSize++] = task;
+        System.out.println("Okay! I have added this:");
+        System.out.printf("\t%s\n", task);
+        System.out.printf("Now you have %d task%s in the list.\n", getListSize(),
```

Comment:
Would it be possible to use the already existing listSize instead of creating a new method getListSize?

Reply:
Thank you for your suggestion! I have implemented this